core/server/src/main/java/alluxio/worker/file/FileDataManager.java

```diff
@@ -69,6 +72,15 @@
   private final Configuration mConfiguration;
   private final Object mLock = new Object();
 
+  /** Provides a per thread rate limiter to throttle async persistence. */
+  private static ThreadLocal<RateLimiter> sPersistenceRateLimiter = new ThreadLocal<RateLimiter>() {
```

Comment:
Since this is thread-local, it only throttles the persistence of the current thread. Should we change it to throttle all the ongoing persistence across threads?